pandas/tests/test_resample.py

```diff
@@ -659,6 +659,21 @@ def f():
                                 'B': {'rb': ['mean', 'std']}})
             assert_frame_equal(result, expected, check_like=True)
 
+    def test_try_aggregate_non_existing_column(self):
```

Comment:
I'm not sure that this test is needed because `test_agg_misc` test covers this case. I added it to show reviewer that original issue is addressed. Let me know and I'll remove this new test.

Reply:
Good to keep it.